src/app/services/news.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable, from } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { News } from '../models/News';
+
+export interface NewsService {
+  getNews(): Observable<News[]>;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DummyNewsService implements NewsService {
+  getNews(): Observable<News[]>  {
+    return from([[
+      {
+        url: 'http://www.google.com',
+        title: 'COVID kills',
+        description: 'Dumb ppl still not wearing masks.'
+      },
+      {
+        url: 'http://www.bridgebase.com/v3',
+        title: 'Moron dies after injecting himself with bleach',
+        description: 'Trump claims injecting bleach as a COVID treatment is "very, very bad"'
+      }
+    ]]);
+  }
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ActualNewsService implements NewsService {
+  url = 'https://api.smartable.ai/coronavirus/news/';
+  subscriptionKey = '955eacf9525e45dd8d46a07b7daa649e';
```

Comment:
Make use of [environment variables](https://angular.io/guide/build#configure-environment-specific-defaults) and just avoid committing those?